cfme/services/myservice/ssui.py

```diff
@@ -80,8 +81,15 @@ def update(self, updates):
 
 
 @MyService.launch_vm_console.external_implementation_for(ViaSSUI)
-def launch_vm_console(self):
+def launch_vm_console(self, catalog_item):
     navigate_to(self, 'VM Console')
+    vm_obj = VM.factory(catalog_item.provisioning_data.get('vm_name') + '001',
```

Comment:
Q: this is unclear to me why you use such vm name. could you please explain ?

Reply:
So when we provision a VM through a service, suppose name of vm is given as test-scat-xRjy, then at the end of provisioning the service will create a VM with name test-scat-xRjy001. If I reran the same service, next VM that will be created will have name test-scat-xRjy002, and so on. In case of this test, I am only creating a single VM by running the service only once, and hence I need to append that 001 to VM name.

Reply:
I woudn't worry about this statement if you added this to test.
However, you are adding this statement to the part of framework.